### ISSUE TITLE
Why the metrics of add and adds are zero?

### ISSUE THREAD

**Issue by aiai84** [ethnhe/PVN3D#62]
Hello, thank you for the nice work of you. For the lackness of enough GPU memory, I set  the batchsize to 10, and evaluate the trained model's add and adds, they are all zero, who can tell me what is the reason. I changed the  batchsize only. Thank you very much!
![2021-01-30 13-39-41屏幕截图](https://user-images.githubusercontent.com/35416507/106348451-5e1ce080-6301-11eb-9276-b0f62757ce98.png)

**Comment by aiai84** [comment]
![2021-01-30 13-39-41屏幕截图](https://user-images.githubusercontent.com/35416507/106348490-9b816e00-6301-11eb-9db2-04a52778fa6c.png)

**Comment by aiai84** [comment]
/home/.conda/envs/pvn3d/lib/python3.6/site-packages/torch/nn/parallel/_functions.py:61: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '
obj_id:  1 0.1 diameter:  0.010209865663
1 ape
***************add auc:  0
***************adds auc:         0
***************add(-s) auc:      0
***************add < 0.1 diameter:       0.0
***************adds < 0.1 diameter:      0.0
=== Training Progress ===
acc_rgbd      --- val: 0.9926   
loss          --- val: 954.2041 
loss_rgbd_seg --- val: 1.3864   
loss_kp_of    --- val: 912.4642 
loss_ctr_of   --- val: 40.3535  
loss_target   --- val: 954.2041

**Comment by ethnhe** [comment]
For testing, we set the batch size to 1 [link](https://github.com/ethnhe/PVN3D/blob/ccc84ba772c9e44ee66505046c8e15ffdf3314b5/pvn3d/common.py#L41). I think the error is not about batch size.
Is the pre-trained model loaded successfully? Check that you put the checkpoint to the corresponding folder and set the ```tst_mdl``` path accordingly. 
```cls='ape'
tst_mdl=train_log/linemod/checkpoints/${cls}/${cls}_pvn3d_best.pth.tar
python3 -m train.train_linemod_pvn3d -checkpoint $tst_mdl -eval_net --test --cls $cls
```
Check the log whether the pre-trained model is successfully loaded when you run the scripts. If it's done successfully, it will print something like " loading from xxx.    **Done.**"

**Comment by aiai84** [comment]
Thank you very much, I tested it once again and it worked!
